versioned_docs/version-2021sp/lecture5.md

```diff
@@ -87,13 +79,14 @@ it in function definition syntax:
 ```tsx title="ContactCard.tsx"
 type Props = { readonly name: string; readonly githubLink: string };
 
-function ContactCard ({ name, githubLink }: Props) = (
-  <div>
-    <div>You can reach to {name} at
+function ContactCard ({ name, githubLink }: Props) = {
```

Comment:
the function syntax is still wrong here. No `=`

Reply:
oops